### PR TITLE
🔀 Merge: Visit Detail Table

### PR DESCRIPTION
**Pull request by hamcheezee**
### Description
This PR introduces new SQL scripts and Python integration to process data from the source database into the `Visit Detail` table and automate the transformation steps.

### Key Changes
- **SQL Scripts:**
   - Path: `../db/sql/postgres/visit_detail/`
   - Purpose: Process records from the source database for the `Visit Detail` table.
- **Python Integration:**
   - Path: `../omop/visit_detail.py`
   - Purpose: Calls the SQL scripts and performs data transformations.
   
### Next Steps
- [x] Map data with other tables: Person, Care_site, Provider, and Visit Occurrence.
- [x] Populate the remaining columns with appropriate values.

### CODE REVIEW

**Comment by hamcheezee** [comment]
Kindly review the final result in omop_sqldev_schema.visit_detail.

File: etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    discharged_to_concept_id,
+    preceding_visit_detail_id,
+    parent_visit_detail_id,
+    visit_occurrence_id
+)
+SELECT
+    -- Assign a unique visit_detail_id to each combination of session_startdate and id
+    ROW_NUMBER() OVER (ORDER by session_startdate, id) AS visit_detail_id,
+    person_id,
+    32037 AS visit_detail_concept_id,
```
Comment (satish-a0) [inline etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql:36]: Added clarification for source from `intra_op__operation` table

File: etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+SELECT
+    -- Assign a unique visit_detail_id to each combination of session_startdate and id
+    ROW_NUMBER() OVER (ORDER by session_startdate, id) AS visit_detail_id,
+    person_id,
+    32037 AS visit_detail_concept_id, 
+    visit_detail_start_date,
+    visit_detail_start_datetime,
+    visit_detail_end_date,
+    visit_detail_end_datetime,
+    32879 AS visit_detail_type_concept_id,
```
Comment (satish-a0) [inline etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql:41]: Clarification added

File: etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    person_id,
+    32037 AS visit_detail_concept_id, 
+    visit_detail_start_date,
+    visit_detail_start_datetime,
+    visit_detail_end_date,
+    visit_detail_end_datetime,
+    32879 AS visit_detail_type_concept_id, 
+    provider_id,
+    care_site_id,
+    'ICU'::text AS visit_detail_source_value,
```
Comment (satish-a0) [inline etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql:44]: Clarification added

Comment (satish-a0) [reply etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql:44]: Ignore operation table

Comment (hamcheezee) [reply etl/pypasar/db/sql/postgres/visit_detail/visit_detail.sql:45]: Updated it.

**Review by satish-a0 — APPROVED** [review]

**Pull request merged**